q2_feature_classifier/tests/test_taxonomic_classifier.py

```diff
@@ -80,7 +80,7 @@ def test_old_sklearn_version(self):
             transformer(input)
 
     def test_old_dirfmt(self):
-        transformer = self.get_transformer(TaxonomicClassifierDirFmt, dict)
+        transformer = self.get_transformer(TaxonomicClassifierDirFmt, Pipeline)
```

Comment:
See my comment about why this change is necessary: https://github.com/qiime2/q2-feature-classifier/pull/50#discussion_r99434658